tdrs-frontend/src/components/Reports/Reports.jsx

```diff
@@ -62,11 +68,27 @@ function Reports() {
     const reportHeader = `${currentStt} - Fiscal Year ${selectedYear} - ${quarters[selectedQuarter]}`
 
     if (form.length === 3) {
+      // Hide upload sections while submitting search
       if (isUploadReportToggled) {
         setIsToggled(false)
       }
 
+      // Clear existing file list from state to ensure fresh results
+      dispatch(clearFileList())
```

Comment:
This is important to prevent files from showing up in the wrong year/quarter after uploading and submitting a new search.